.github/CODEOWNERS

```diff
@@ -1,10 +1,10 @@
 # Users referenced in this file will automatically be requested as reviewers for PRs that modify the given paths.
 # See https://help.github.com/articles/about-code-owners/
 
-/global.json                                   @aspnet/build
-/azure-pipelines.yml                           @aspnet/build
-/eng/                                          @aspnet/build
-/eng/common/                                   @dotnet-maestro-bot @dougbu
-/eng/Versions.props                            @dotnet-maestro-bot @dougbu
-/eng/Version.Details.xml                       @dotnet-maestro-bot @dougbu
-/src/Razor/                                    @dotnet/razor-tooling @NTaylorMullen @ryanbrandenburg @allisonchou @sharwell
+/global.json                                   @dotnet/razor-tooling
+/azure-pipelines.yml                           @dotnet/razor-tooling
+/eng/                                          @dotnet/razor-tooling
+/eng/common/                                   @dotnet-maestro-bot
+/eng/Versions.props                            @dotnet-maestro-bot
+/eng/Version.Details.xml                       @dotnet-maestro-bot
+/src/                                          @dotnet/razor-tooling
```

Comment:
We still want to mention us directly for the src path: `@dotnet/razor-tooling @NTaylorMullen @ryanbrandenburg @allisonchou` because otherwise we don't get fully notified for those 😄 


<!--StartFragment-->@dotnet/razor-toolingWrite&nbsp;<!--EndFragment-->

 @NTaylorMullen @ryanbrandenburg @allisonchou

Reply:
Huh❔ Is there a bug somewhere causing GitHub to skip sending notifications to everyone on the listed team❔

Reply:
Probably just email outlook rules

Reply:
Oh wait actually no, it's thatonce one person reviews a PR it removes the rest of the people in the group. It's a GitHub feature which is a bit less than ideal

Reply:
```suggestion
/src/                                          @dotnet/razor-tooling @NTaylorMullen @ryanbrandenburg @allisonchou @sharwell
```